samples/src/main/java/com/example/bigquery/ListModels.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQuery.ModelListOption;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Model;
+
+// [START bigquery_list_models]
+public class ListModels {
+
+  public static void runListModels() {
+    // TODO(developer): Replace these variables before running the sample.
+    String datasetName = "MY_DATASET_NAME";
+    listModels(datasetName);
+  }
+
+  public static void listModels(String datasetName) {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      try {
+        Page<Model> models = bigquery.listModels(datasetName, ModelListOption.pageSize(100));
+        models
+            .iterateAll()
+            .forEach(
+                model ->
+                    System.out.println(
+                        model.getModelId() + " models in dataset listed successfully."));
```

Comment:
This doesn't look right - this is printing as `"$ModelId models in dataset listed successfully"`.

I think you want probably want to do is print something like `Model: modelId = $modelId, modelName = $modelName

Reply:
Looks like there's only getModelId method to get the model Id. I think I will leave it this way but update to just "model" (instead of "models").